app.py

```diff
@@ -10,11 +13,23 @@
 
 @app.route('/emoji', methods=['GET'])
 def get_emoji_list():
+
     try:
-        with open('results.json') as json_file:
-            data = json.load(json_file)
-            return jsonify(data)
+
+        source_client = emoji_client(os.environ["SOURCE_SLACK_API_TOKEN"])
+        source_dict = source_client.emoji_dict()
+
+        # map alias values to URL values or None
+        names_and_urls = {}
+        for k, v in source_dict.items():
+            names_and_urls[k] = url_even_if_alias(source_dict, k)
+
+        # filter out None values (aliases of standard emoji).
+        names_and_urls = { k: v for k, v in names_and_urls.items() if v is not None and not v.startswith("alias:") }
```

Comment:
It seems like aliases of slack's built-in aliases stick around, pushing this up for now.  Fine to limit the UI options on selecting aliases, those are one-step processes in Slack's UI.